macros/staging/hash_columns.sql

```diff
@@ -70,7 +70,7 @@
 
             {{-
                 dbtvault.hash(
-                    columns=src_col_config if src_col_config is string else src_col_config['columns'],
+                    columns=src_col_config['columns'] if src_col_config is mapping else src_col_config,
```

Comment:
turns out the `is string` should be `is string or is list` so I flipped this if condition to test for mapping and extract the columns or just  pass the columns (string or list) down otherwise

Reply:
As far as I know there's no concise way to test something is a list other than `is interable and is not string`, which is a massive pain. Was this a typo or have you discovered a way? I don't see this in your code.

Reply:
I haven't discovered a way, it's also not a typo.

if you look up in the if condition you can see I'm doing additional checks which means some things are guaranteed to be true when I get down here